docs/reference/cat/indices.asciidoc

```diff
@@ -1,125 +1,101 @@
 [[cat-indices]]
 === cat indices
 
-The `indices` command provides a cross-section of each index.  This
-information *spans nodes*. For example:
+Returns high-level information about indices in a cluster.
 
-[source,js]
---------------------------------------------------
-GET /_cat/indices/twi*?v&s=index
---------------------------------------------------
-// CONSOLE
-// TEST[setup:huge_twitter]
-// TEST[s/^/PUT twitter2\n{"settings": {"number_of_replicas": 0}}\n/]
 
-Might respond with:
+[[cat-indices-api-request]]
+==== {api-request-title}
 
-[source,txt]
---------------------------------------------------
-health status index    uuid                   pri rep docs.count docs.deleted store.size pri.store.size
-yellow open   twitter  u8FNjxh8Rfy_awN11oDKYQ   1   1       1200            0     88.1kb         88.1kb
-green  open   twitter2 nYFWZEO7TUiOjLQXBaYJpA   1   0          0            0       260b           260b
---------------------------------------------------
-// TESTRESPONSE[s/\d+(\.\d+)?[tgmk]?b/\\d+(\\.\\d+)?[tgmk]?b/]
-// TESTRESPONSE[s/u8FNjxh8Rfy_awN11oDKYQ|nYFWZEO7TUiOjLQXBaYJpA/.+/ non_json]
+`GET /_cat/indices/{index}`
 
-We can tell quickly how many shards make up an index, the number of
-docs, deleted docs, primary store size, and total store size (all shards including replicas).
-All these exposed metrics come directly from Lucene APIs.
 
-*Notes:*
+[[cat-indices-api-desc]]
+==== {api-description-title}
 
-1. As the number of documents and deleted documents shown in this are at the lucene level,
-it includes all the hidden documents (e.g. from nested documents) as well.
+You can use the cat indices API to get the following information for each index
+in a cluster:
```

Comment:
I'd probably drop "You can" and just use the imperative.

Reply:
Changed with babd966. Thanks!